### DISCUSSION TITLE
Inquiry on How to Contact the Scoop.sh Team for Open Source Study

### DISCUSSION THREAD

**Discussion by RPELng** [ScoopInstaller/Scoop#6175] [Help and Questions] [answered]
Hello Scoop Team,

I hope you're doing well. I'm currently working on an analysis of various solutions similar to Scoop.sh as part of a study for a major client. As an open-source company, we prefer to prioritize and deepen our knowledge of solutions that are fully open-source, like Scoop, over proprietary alternatives.

I have some questions regarding future developments and possible evolutions of Scoop, but I haven’t found any other way to contact your team directly. Could you please guide me on how best to get in touch with someone on the team for further discussion? Whether it's via email, Slack, or any other preferred channel, I’d appreciate the information.

Thank you for your time and support. I look forward to hearing from you and learning more about Scoop's future!

Best regards,

**Comment by HUMORCE** [comment]
Are the members of ScoopInstaller team contacted to each other out of GH? At least I don't.

I think this is a scam with empty GH account, you did not even leave information about you.

**Comment by RPELng** [comment, marked as answer]
My analytical study on the subject has since been completed so sorry it is no longer an open subject on my side.
